ext/pdo/tests/pdo_039.phpt

```diff
@@ -29,6 +34,9 @@ if ($pass === false) $pass = NULL;
 $conn = new PDO($dsn, $user, $pass, $attr);
 
 $query = 'SELECT 1';
+if ($conn->getAttribute(PDO::ATTR_DRIVER_NAME) === 'oci') {
```

Comment:
This is breaking the following `errorCode()` assertion, is that expected side effect of `getAttribute()` method call?

Reply:
fixed in https://github.com/php/php-src/commit/d86d60ef6693cc2dd6e909e65d105333ffcbcafa

Reply:
But the expected output is then broken for Sqlite and 2nd call to `errorCode()` does not return a null! Why?

I thought this is how the `errorCode()` should work - return `string(5) "00000"` for the 1st invocation after some PDO operation, but then return `null` when invoken for 2nd (3rd, ...) time.

Any idea why and how to fix it?